numpy/core/tests/test_deprecations.py

```diff
@@ -400,6 +400,36 @@ def test_fortran_contiguous(self):
         self.assert_deprecated(np.ones((2,2)).T.view, args=(np.int8,))
 
 
+class TestInvalidOrderParameterInputForFlattenArrayDeprecation(_DeprecationTestCase):
+    """Invalid arguments to the ORDER parameter in array.flatten() should not be
+    allowed and should raise an error.  However, in the interests of not breaking
+    code that may inadvertently pass invalid arguments to this parameter, a
+    DeprecationWarning will be issued instead for the time being to give developers
+    time to refactor relevant code.
+    """
+
+    def test_flatten_array_non_string_arg(self):
+        x = np.zeros((3, 5))
+        self.message = "Non-string object detected for " + \
```

Comment:
PEP8 prefers putting the whole string inside parenthesis, and getting rid of both the `\`s and the `+`s, relying on [string literal concatenation](https://docs.python.org/2/reference/lexical_analysis.html#string-literal-concatenation).